charts/citylens-routes-ui/values.yaml

```
@@ -0,0 +1,185 @@
+# @param env.MAPGL_API_URL Map API base URL
+# @param env.MAPGL_COPYRIGHT_VARIANT Copyright variant, can be '2gis', 'urbi' or empty
+# @param env.MAPGL_KEY API key for mapgl
+# @param env.MAPGL_STYLE_ID Map style ID
+# @param env.ROUTES_API_URL Backend (citylens-routes-api) base URL
+env:
+  CATALOG_API_URL: ''
+  MAPGL_API_URL: ''
+  MAPGL_COPYRIGHT_VARIANT: ''
+  MAPGL_KEY: ''
+  MAPGL_STYLE_ID: ''
+  ROUTES_API_URL: ''
+
+# @section Common deployment settings
+
+# @param replicas A replica count for the pod.
+# @param revisionHistoryLimit Revision history limit (used for [rolling back](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) a deployment).
+# @param terminationGracePeriodSeconds Seconds pod needs to [terminate](https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods) gracefully
+# @param nodeSelector Kubernetes [node selectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
+# @param affinity Kubernetes pod [affinity settings](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
+# @param tolerations Kubernetes [tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) settings.
+# @param podAnnotations Kubernetes [pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
+# @param podLabels Kubernetes [pod labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+# @param annotations Kubernetes [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).
+# @param labels Kubernetes [labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/).
+# @param readinessProbe.enabled Enable [readinessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) on PRO UI containers
+# @param livenessProbe.enabled Enable [livenessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes) on PRO UI containers
+# @param livenessProbe.healthcheckPath Application http path for health check
+# @param containerPort Port on which application listen connection in container
+replicas: 1
+revisionHistoryLimit: 3
+terminationGracePeriodSeconds: 60
+nodeSelector: {}
+affinity: {}
+tolerations: []
+podAnnotations: {}
+podLabels: {}
+annotations: {}
+labels: {}
+readinessProbe:
+  enabled: false
+livenessProbe:
+  enabled: true
+  healthcheckPath: /health
+containerPort: 3000
+
+# @section Logs
+
+# @param log.errorLog.level Error log level. Allowed values: `debug`, `info`, `notice`, `warn`, `error`, `crit`, `alert`, `emerg`.
+# @param log.accessLog NGINX access log definition.
+# @param log.formats List of log formats to be used in NGINX configuration
+log:
+  formats: []
+  errorLog:
+    level: debug
+  accessLog: off
```

чтобы линтер не ругался
```suggestion
  accessLog: 'off'
```

---

Сделал `/dev/null`